pom.xml

```diff
@@ -2368,7 +2368,8 @@
                     <groupId>org.sonatype.plugins</groupId>
                     <artifactId>nexus-staging-maven-plugin</artifactId>
                     <version>${dep.nexus-staging-plugin.version}</version>
-                    <extensions>true</extensions>
+                    <!-- This plugin is not registered as an extension because it will fail to activate
```

Comment:
Maven 3 supports plugins which alter the build itself, i.e., instead of specifying a plugin with a phase or goal, you can simply mark a plugin as an "extension" and it will automatically register its own phases, goals, configuration, and even modify the configuration of other plugins or dependencies.

Previously, we were using the extension mechanism, which meant less configuration in the pom, but we can't do that anymore because of how this plugin interactives with the provisio plugin.  So instead, we're configuring it manually.